### DISCUSSION TITLE
How can I close the bell?

### DISCUSSION THREAD

**Discussion by Straho-Chen** [JetBrains/ideavim#479] [Q&A] [answered]
I want to close the error bell, but when I try "set noeb" or "set noerrorbells" I find that it cannot recognize it. So, how can I close the bell?

**Comment by citizenmatt** [comment, marked as answer]
You can set the `'visualbell'` option - `:set visualbell`. Vim uses this to replace the beep with a flash of the screen. IdeaVim doesn't flash the screen, but it will stop the beep. (`'errorbells'` only stops the beep for errors, not the normal usages such as hitting Escape in normal mode. `'belloff'` might be a better option to fine tune what does or doesn't beep, but IdeaVim doesn't support it)

**Reply by Straho-Chen** [reply]
Thanks!
